### PR TITLE
Updated CR

### PR DESCRIPTION
**Pull request by alexeykudinkin**
Subject.

### CODE REVIEW

**Comment by mkaput** [comment]
Why?

**Comment by alexeykudinkin** [comment]
@mkaput please, clarify your question.

**Comment by mkaput** [comment]
I do not understand:
 - why the website is being moved under JetBrains flag - especially as I am it's original creator. I can understand transferring project's copyrights itself as it's primary work of JB employees during their work time (though I believe copyrights are also owned by all contributors, as we haven't signed any CLAs), and I have no objectives for this. But the website is different story
 - and why should copyrights hold only for years 2015-2016 and not 2017?

**Comment by alexeykudinkin** [comment]
Oh, sorry about that! 

Just blindly copy-pasted this, from the project side. Will update it properly.

Point of consideration though is that, Alexey works as FTE for JetBrains and works on this project full-time making contributions both to the plugin itself, as well as this website (contrary, i, while still being FTE at JetBrains _never_ worked on this project as such, that's why being listed as an individual contributor). 

I'm no longer the JetBrains employee, though i was contacted by JetBrains representatives that raised somewhat legitimate concerns that the company (while indeed supporting the project) weren't mentioned as the CR holder.

**Comment by alexeykudinkin** [comment]
And related to years: year basically means the _start_ of attribution of the CR and doesn't assume any yield of it in case not specified otherwise.

**Comment by mkaput** [comment]
I believe that actual phrase was fine. Maybe something like `JetBrains & IntelliJ Rust Team` or `JetBrains & IntelliJ Rust Open Source Team`

**Comment by alexeykudinkin** [comment]
Unfortunately there's no such a legal entity as Intellij Rust Open Source team, that's why it's more accurate to list it as the original authors, and broadly defined group of other _contributors_.

**Comment by alexeykudinkin** [comment]
That doesn't mean though that those particular contributors cede the CR having their contributions in the code base.

**Comment by mkaput** [comment]
Hmm, ok I get. But one thing according to the last change: website was made in 2016 ;)

File: _includes/footer.html [hunk 1]
```diff
@@ -6,7 +6,8 @@ <h2 class="footer-heading">{{ site.title }}</h2>
 
     <div class="footer-col-wrapper">
       <div class="footer-col footer-col-1">
-        <p>&copy; 2016-2017 IntelliJ Rust Team</p>
+        <p>&copy; 2016 Marek Kaput and contributors</p>
+        <p>&copy; 2016 JetBrains</p>
```
Comment (mkaput) [inline _includes/footer.html]: And maybe why not merge this?

```html
<p>&copy; 2016-2017 JetBrains, Marek Kaput and contributors</p>
```

**Comment by mkaput** [comment]
But do not merge this yet. Let's wait for @matklad's opinion

**Comment by matklad** [comment]
:+1:

**Pull request merged**